### PR TITLE
Create php.yml

### PR DESCRIPTION
**Pull request by timothygwebb**

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
       uses: actions/cache@v2
       with:
         path: vendor
-        key: ${{ runner.os }}-php-${{ hashFiles('boxbilling/src/composer.lock') }}
+        key: ${{ runner.os }}-php-${{ hashFiles('composer.lock') }}
```
Comment (samapraku) [inline .github/workflows/php.yml:25]: Have you tried this? 
`'./src/composer.lock'`

**Comment by samapraku** [comment]
@timothygwebb You need to specify the work-directory keyword for each command. I have managed it to build successfully to the point you have.

```yaml
name: PHP Composer

on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]

jobs:
  build:

    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v2

    - name: Validate composer.json and composer.lock
      run: composer validate
      working-directory: ./src

    - name: Cache Composer packages
      id: composer-cache
      uses: actions/cache@v2
      with:
        path: ./src/bb-vendor
        key: ${{ runner.os }}-php-${{ hashFiles('composer.lock') }}
        restore-keys: |
          ${{ runner.os }}-php-    
    - name: Install dependencies
      if: steps.composer-cache.outputs.cache-hit != 'true'
      run: composer install --prefer-dist --no-progress --no-suggest
      working-directory: ./src

    # Add a test script to composer.json, for instance: "test": "vendor/bin/phpunit"
    # Docs: https://getcomposer.org/doc/articles/scripts.md

    # - name: Run test suite
    #   run: composer run-script test
    #     working-directory: ./src
```

**Comment by samapraku** [comment]
@timothygwebb I thought the goal was to make this file trigger and run unit tests not so?

The bottom section contains the command for running the unit test. I will confirm the correct code to use and update here

**Comment by timothygwebb** [comment]
@samapraku I was going to use ant install like Travis does and discuss what test to add before adding. But it does check the dependencies of composer now if you look in the details of the action.

**Comment by samapraku** [comment]
@timothygwebb  I have already prepared the actions that runs the unit tests in this pull request. https://github.com/boxbilling/boxbilling/pull/695

It is able to run the tests and I need to add the actions to run the other tests.

I think you can save your time from working on this.

**Pull request merged**